InvenTree/InvenTree/static/fullcalendar/main.js

```diff
@@ -13974,7 +13974,7 @@ var FullCalendar = (function (exports) {
             }
             return null;
         },
-        fetch: function (arg, onSuccess, onFailure) {
```

Comment:
This file is from a third-party library, we should leave this one alone. Please revert changes in this file :)